app/models/cart.rb

```diff
@@ -43,6 +43,7 @@ class Cart < FlexCommerceApi::ApiBase
     has_one :shipping_address, class_name: "::FlexCommerce::Address"
     has_one :billing_address, class_name: "::FlexCommerce::Address"
     has_one :shipping_method, class_name: "::FlexCommerce::ShippingMethod"
+    has_one :free_shipping_promotion, class_name: "::FlexCommerce::FreeShippingPromotion"
```

Comment:
It's a different object though in the API - as it's not listed with the other promotions.